tests/UtilTest.php

```diff
@@ -79,4 +79,44 @@ public function testShorten(array $classes, array $expected): void
     {
         $this->assertEquals($expected, Util::shorten($classes));
     }
+
+    public function convertEnumsFixtures(): iterable
+    {
+        return [
+            [\OpenApi\Tests\Fixtures\PHP\StatusEnumBacked::class, [1, 2, 3]],
```

Comment:
Could you please use `use` statements and import those enums rather than using FQDN?